tools/dds/dds-sniffer/CMakeLists.txt

```diff
@@ -4,18 +4,19 @@
 
 project(rs-dds-sniffer)
 
-set(SRC dds-sniffer.cpp)
-set(INC dds-sniffer.hpp)
+set(SRC rs-dds-sniffer.cpp ${REPO_ROOT}/include/librealsense2/dds/dds-participant.cpp)
```

Comment:
Please try to use `dds-client.cmake`

Reply:
Using. Also updated dds-client.cmake to include easylogging